assets/javascripts/modules/checkout/email-check.js

```diff
@@ -17,17 +20,17 @@ define([
                     url: '/checkout/check-identity?email=' + email
                 }).then(function (response) {
                     if (response.emailInUse) {
-                        var msg = 'Your email is already in use! Please sign in or use another email address';
-                        reject(new Error(msg));
+                        reject(Error('EMAIL_IN_USE'));
```

Comment:
Not actually sure why you are using an `Error` object unless you specifically want the stack trace. Probably still need `new` here though.